governance/contributor-ladder.md

```diff
@@ -0,0 +1,111 @@
+# Outlining your Contributor Path
+
+The CNCF Environmental Sustainability TAG aims to support individuals who are passionate about cloud native sustainability and provide a space for personal growth as contributors.
+This document outlines a path for advancing as a contributor in the TAG. 
+It's important to note that your journey may not precisely align with this path, as the document serves as a general guide.
+If you have already made significant contributions to another TAG or a project within the CNCF landscape and are familiar with the community, your path will naturally differ from that of someone who is just starting out.
+
+- [Outlining your Contributor Path](#outlining-your-contributor-path)
+  - [Starting out](#starting-out)
+  - [Taking ownership \& responsibility](#taking-ownership--responsibility)
+  - [Leading a project](#leading-a-project)
+  - [Helping out](#helping-out)
+  - [Leading a Working Group](#leading-a-working-group)
+  - [Nominated as Tech Lead](#nominated-as-tech-lead)
+  - [Nominated as Co-Chair](#nominated-as-co-chair)
+
+## Starting out
+
+When venturing into the open source space, it's crucial to familiarize yourself with the way we operate and collaborate as a global community.
+Take some time to reflect on your personal journey.
+Find out if you genuinely derive joy from this line of work and consider what areas of knowledge you are particularly curious about.
+Remember, actively participating and providing valuable input are significant aspects of your involvement in the community.
+By engaging with others and sharing your insights, you contribute to the collective growth and success of the open source ecosystem.
+
+* Join Community Meetings, the Slack Channel and the mailing list (take a look at [this page](https://github.com/cncf/tag-env-sustainability/#contact))
+* Proactively participate in discussions at community meetings, in Slack threads, or [GitHub issues](https://github.com/cncf/tag-env-sustainability/issues).
+* Reflect on your personal journey spent at TAG
```

Comment:
I'd remove the "reflect" one.  I'm not sure what it adds here.

Reply:
Since this is the first step in the journey, people don't know if they like the subject, the group, the foundation, the way we work, and so on. This item mentions that you should think about whether the way we work suits you. 

I will remove it as it is not a clear action. And ... You will, of course, move away from the group if it does not suit you ....